test/api/liskApi.js

```diff
@@ -896,14 +903,7 @@ describe('Lisk.api()', function () {
 	describe('#checkReDial', function () {
 
 		it('should check if all the peers are already banned', function () {
-			(lisk.api().checkReDial()).should.be.equal(true);
-		});
-
-		it('should be able to get a new node when current one is not reachable', function (done) {
```

Comment:
Why is this test removed?

Reply:
Because it duplicates this test: https://github.com/LiskHQ/lisk-js/pull/548/files#diff-03dab7dc7afe96c8322cab28bdf86f72R940